test/e2e/e2e.go

```diff
@@ -276,3 +284,18 @@ func runKubernetesServiceTestContainer(c clientset.Interface, ns string) {
 		e2elog.Logf("Output of clusterapi-tester:\n%v", logs)
 	}
 }
+
+// getClusterIPFamily obtains the (primary) ip family of the cluster based
+// on the Cluster IP address of the default kubernetes service
+func getClusterIPFamily(c clientset.Interface) string {
```

Comment:
please add a TODO here to detect dual stack too.
i saw the discussion in #dual-stack.

Reply:
I added the TODO in L151 because I don´t think that detecting dual-stack should be done in this function. 
Actually, I have doubts about the needs from current e2e tests to use that information, that´s why I think that´s better to separate (primary) ip family detection from dual or single stack family.